docs/Configuration.md

```diff
@@ -0,0 +1,106 @@
+
+Each runtime defines an `appsettings.{Environment}.json` for each target environment. This file should contain environment specific configuration. In addition to the specific environment configuration files there should also exist a `appsettings.json`. This file should contain cross cutting environment configuration. The `appsettings.{Environment}.json` configuration takes presence over the `appsettings.json` configuration, meaning - if the configuration key `foo` exists in both files, the value from `appsettings.{Environment}.json` is used. 
+
+## 2.1. Word of caution
+Due to the implicit and cross cutting nature of `appsettings.json` **it should be used cautiously.** It's often better to let a runtime fail hard and crash due to no configuration, than using the wrong configuration value silently. Say for example that a `.well-known` authentication service endpoint is configured in `appsettings.json` because the same endpoint is used in `Development`, `Test` and `Stage`. However, the developer forgets to add it to `appsettings.Production.json` and thereby not overriding it for the `Production` environment. Thus `Production` is now using the same `.well-known` endpoint as the rest of the environments and thereby open to non-production tokens. It should produce a **big fat red critical error**, but instead it continues on like everything is fine. Therefore the `.well-known` endpoint, and many other configurations, does **not belong** in `appsettings.json`. **The absence of a configuration is often better than the wrong one silently working.** 
+
+## 2.2. How to
+Adding, removing or updating non secret configuration is as simple as modifying `appsettings.json` for cross cutting configuration or `appsettings.[Environment].json` for environment specific configuration. **Use caution when adding configuration to `appsettings.json`**, see [2.1. Word of caution](#21-word-of-caution). When adding a required configuration to `appsettings.[Environment].json`, remember to add the configuration for all the environment specific json files. GitHub actions will do the rest when merging to main.
+
+## 2.3. CI/CD
+When a pull request containing configuration changes is merged with main, one of two things happens:
+- If there are configuration changes **and not** changes to the `src` folder, a GitHub action workflow kicks in pushing **only** the configuration to Azure App Configuration and updating the sentinel value. 
+- If there are changes to the `src` folder regardless of configuration changes, the normal build-deploy GitHub action workflow will kick in. It in turn will reference the above workflow pushing changes to Azure App Configuration and updating the sentinel value **only** when there are configuration changes. We can use the [paths-filter](https://github.com/dorny/paths-filter) to run the config action depending on if there are changes to the configuration. 
```

Comment:
Bare en språklig greie - foreslår vi endrer "We can use the paths-filter ..." til "We use the paths-filter ...", så dette fungerer som dokumentasjon for hvordan det faktisk gjøres og ikke hvordan det potensielt kan gjøres :)